### DISCUSSION TITLE
Logging during training

### DISCUSSION THREAD

**Discussion by parthnatekar** [tensorflow/compression#104] [General]
Hi, 

Is there a way to track my losses and variables during training? Can't find an option to do that - I'm used to TF2/Keras so if you have any resources where I can find how to do that while running the TF1 session please share them.

**Comment by jonaballe** [comment]
Hi, yes, you can use a [`Mean` metric](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/Mean), or [summaries](https://www.tensorflow.org/api_docs/python/tf/summary), for example. In our example models we are tracking rate, distortion and loss value using metrics. The `Mean` metric averages a given scalar value over the epoch, and it can be logged to TensorBoard using the [TensorBoard callback](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard). Alternatively, you can directly create a summary which will log the value instantaneously.
